Quaver/Program.cs

```diff
@@ -111,7 +112,11 @@ private static void Run()
                 Logger.Error(e, LogType.Runtime);
             }
 
+#if VISUAL_TESTS
+            using (var game = new QuaverGame(new HotLoader("../../../../Quaver.Shared/")))
```

Comment:
Is there a better way? This will break with testing published builds.

Reply:
How? And why would you test with a published build in the first place?

Reply:
> How?

Maybe there's some project variable with source location or something? Should at least make it overridable via an environment variable.

> And why would you test with a published build in the first place?

There already are some things that for some unknown reason work only in the published build (IPC). So I guess that's my reasoning. Even though you probably won't need IPC when building a UI through the visual tests.